Resources/Prototypes/Catalog/Fills/Crates/fun.yml

```
@@ -514,4 +557,32 @@
\ No newline at end of file
+        prob: 0.05
+
+- type: entity
+  id: CrateFunSelfInsertPlushie # Self insert crate :-)
+  parent: CrateGenericSteel
+  name: crew plushie crate
+  description: A box full of cute plushies of beloved crewmembers.
+  components:
+  - type: StorageFill
+    contents:
+      - id: PlushieBenny
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)